demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -898,7 +1216,9 @@ static int run_quic_server(SSL_CTX *ctx, int fd)
         /*
          * Free the connection, then loop again, accepting another connection.
          */
-        SSL_free(conn);
+        close_all_ids(&h3ssl);
+        SSL_free(h3ssl.ssl_ids[1].s);
```

Comment:
ssl_ids[1] (connection) and ssl_ids[0] (listener) are special I would suggest to define macros LISTENER, CONNECTION to distinguish those two objects in ids array.

Reply:
I have arranged the logic.